### DISCUSSION TITLE
How to make firefox equal to the preview image?

### DISCUSSION THREAD

**Discussion by DiegoSof2** [prasanthrangan/hyprdots#85] [Q&A] [answered]
How do firefox stick to this theme

![image](https://github.com/prasanthrangan/hyprdots/assets/105827698/4ef1c65d-39f4-4b6b-9c2a-5b2f763d87a7)

**Comment by prasanthrangan** [comment, marked as answer]
Hi, please install `nightTab` extension and import this [t2_nightTab.json](https://github.com/prasanthrangan/hyprdots/blob/main/Source/misc/t2_nightTab.json) file.

**Reply by DiegoSof2** [reply]
Thanks for helping
